### DISCUSSION TITLE
Commits ridiculos para pasar los checks del codacy

### DISCUSSION THREAD

**Discussion by MarioArocaPaez** [perealberto/decide-single-zambrano#73] [Q&A]
Chicos en mi PR el codacy me está dando problemas y quejandose por todo, creo que ya lo tengo todo como quiere menos por la última cosa y es que se me está quejando por un espacio en blanco de mas
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/b70612f3-05f1-4f54-a4f7-6d72cf4b73d0)
Creeis que deberia hacer un commit solo para quitar ese espacio en blanco o mergeamos ignorando el failed del codacy?

**Comment by silvglad** [comment]
Como te comenté, mientras que no sea un problema de tamaño medio avisado por Codacy, creo que no merece la pena realizar commits para arreglar desperfectos solo relacionados al typing/estilo. Por lo general, debemos intentar que nuestros commits sean atómicos, por lo que se podrían arreglar en el futuro por un rama de hotfix dedicada a las incidencias avisadas por Codacy, aunque no lo veo completamente necesario para nuestro proyecto y futura entrega

**Comment by laurolmer** [comment]
Podemos implementar ESLint u otra herramienta para que esos fallos tontorrones los solucione automáticamente y así evitar que Codacy lance advertencias que dicten con que no cumplimos con los estándares ideales de gestión del código, no sé cómo lo veis.
Alberto estaba revisando justo ayer la herramienta.
